### ISSUE TITLE
Release v0.0.2 has errors which are fixed in master

### ISSUE THREAD

**Issue by sougat818** [Mongey/terraform-provider-confluentcloud#24]
Has anyone else faced this or I am doing something wrong. Got this when I tried the example given in the readme. Environment is created successfully. 

```confluentcloud_service_account.test: Creating...
confluentcloud_service_account.test: Creation complete after 0s [id=90607]
confluentcloud_environment.environment: Creating...
confluentcloud_environment.environment: Creation complete after 1s [id=env-v13r0]
confluentcloud_kafka_cluster.test: Creating...

Error: rpc error: code = Unavailable desc = transport is closing


panic: interface conversion: interface {} is nil, not string
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2:
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: goroutine 59 [running]:
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: github.com/Mongey/terraform-provider-confluent-cloud/ccloud.clusterCreate(0xc0000faf50, 0x1b87120, 0xc000564fa0, 0x2, 0x2538700)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/var/folders/2k/fy8w13m902j3twpw64bp_wbc0000gn/T/=tf-installer.XOE99qcq/tf-installer-clone-confluentcloud/ccloud/resource_kafka_cluster.go:115 +0x62d
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: github.com/hashicorp/terraform/helper/schema.(*Resource).Apply(0xc000044b00, 0xc0002a0fa0, 0xc0001246a0, 0x1b87120, 0xc000564fa0, 0x1a7b801, 0xc00078cae8, 0xc000796990)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/github.com/hashicorp/terraform@v0.12.1/helper/schema/resource.go:286 +0x365
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: github.com/hashicorp/terraform/helper/schema.(*Provider).Apply(0xc000044d00, 0xc0007df8d0, 0xc0002a0fa0, 0xc0001246a0, 0xc00079a088, 0xc0001166e0, 0x1a7d700)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/github.com/hashicorp/terraform@v0.12.1/helper/schema/provider.go:285 +0x99
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: github.com/hashicorp/terraform/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc000010a58, 0x1dffb40, 0xc000789d10, 0xc00010e420, 0xc000010a58, 0xc000789d10, 0xc0004bca48)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/github.com/hashicorp/terraform@v0.12.1/helper/plugin/grpc_provider.go:842 +0x892
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: github.com/hashicorp/terraform/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x1b893e0, 0xc000010a58, 0x1dffb40, 0xc000789d10, 0xc0002a0dc0, 0x0, 0x1dffb40, 0xc000789d10, 0xc00021a5a0, 0x1c3)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/github.com/hashicorp/terraform@v0.12.1/internal/tfplugin5/tfplugin5.pb.go:3019 +0x217
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: google.golang.org/grpc.(*Server).processUnaryRPC(0xc000498f00, 0x1e09f40, 0xc000642a80, 0xc000791000, 0xc0006563f0, 0x24fed00, 0x0, 0x0, 0x0)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/google.golang.org/grpc@v1.18.0/server.go:966 +0x46a
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: google.golang.org/grpc.(*Server).handleStream(0xc000498f00, 0x1e09f40, 0xc000642a80, 0xc000791000, 0x0)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/google.golang.org/grpc@v1.18.0/server.go:1245 +0xc9e
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc000038570, 0xc000498f00, 0x1e09f40, 0xc000642a80, 0xc000791000)
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/google.golang.org/grpc@v1.18.0/server.go:685 +0xa1
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: created by google.golang.org/grpc.(*Server).serveStreams.func1
2020-07-20T17:17:09.130+1000 [DEBUG] plugin.terraform-provider-confluentcloud_v0.0.2: 	/Users/sougata/go/pkg/mod/google.golang.org/grpc@v1.18.0/server.go:683 +0xa1
2020/07/20 17:17:09 [DEBUG] confluentcloud_kafka_cluster.test: apply errored, but we're indicating that via the Error pointer rather than returning it: rpc error: code = Unavailable desc = transport is closing
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalMaybeTainted
2020/07/20 17:17:09 [TRACE] EvalMaybeTainted: confluentcloud_kafka_cluster.test encountered an error during creation, so it is now marked as tainted
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalWriteState
2020/07/20 17:17:09 [TRACE] EvalWriteState: removing state object for confluentcloud_kafka_cluster.test
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalApplyProvisioners
2020/07/20 17:17:09 [TRACE] EvalApplyProvisioners: confluentcloud_kafka_cluster.test has no state, so skipping provisioners
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalMaybeTainted
2020/07/20 17:17:09 [TRACE] EvalMaybeTainted: confluentcloud_kafka_cluster.test encountered an error during creation, so it is now marked as tainted
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalWriteState
2020/07/20 17:17:09 [TRACE] EvalWriteState: removing state object for confluentcloud_kafka_cluster.test
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalIf
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalIf
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalWriteDiff
2020-07-20T17:17:09.132+1000 [DEBUG] plugin: plugin process exited: path=/Users/sougata/.terraform.d/plugins/terraform-provider-confluentcloud_v0.0.2 pid=90374 error="exit status 2"
2020/07/20 17:17:09 [TRACE] <root>: eval: *terraform.EvalApplyPost
2020/07/20 17:17:09 [ERROR] <root>: eval: *terraform.EvalApplyPost, err: rpc error: code = Unavailable desc = transport is closing
2020/07/20 17:17:09 [ERROR] <root>: eval: *terraform.EvalSequence, err: rpc error: code = Unavailable desc = transport is closing
2020/07/20 17:17:09 [TRACE] [walkApply] Exiting eval tree: confluentcloud_kafka_cluster.test
2020/07/20 17:17:09 [TRACE] vertex "confluentcloud_kafka_cluster.test": visit complete
2020/07/20 17:17:09 [TRACE] dag/walk: upstream of "provider.confluentcloud (close)" errored, so skipping
2020/07/20 17:17:09 [TRACE] dag/walk: upstream of "meta.count-boundary (EachMode fixup)" errored, so skipping
2020/07/20 17:17:09 [TRACE] dag/walk: upstream of "root" errored, so skipping
2020/07/20 17:17:09 [TRACE] statemgr.Filesystem: have already backed up original terraform.tfstate to terraform.tfstate.backup on a previous write
2020/07/20 17:17:09 [TRACE] statemgr.Filesystem: state has changed since last snapshot, so incrementing serial to 13
2020/07/20 17:17:09 [TRACE] statemgr.Filesystem: writing snapshot at terraform.tfstate
2020/07/20 17:17:09 [TRACE] statemgr.Filesystem: removing lock metadata file .terraform.tfstate.lock.info
2020/07/20 17:17:09 [TRACE] statemgr.Filesystem: unlocking terraform.tfstate using fcntl flock
2020-07-20T17:17:09.143+1000 [DEBUG] plugin: plugin exited



!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Terraform crashed! This is always indicative of a bug within Terraform.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Terraform[1] so that we can fix this.

When reporting bugs, please include your terraform version. That
information is available on the first line of crash.log. You can also
get it by running 'terraform --version' on the command line.

SECURITY WARNING: the "crash.log" file that was created may contain
sensitive information that must be redacted before it is safe to share
on the issue tracker.

[1]: https://github.com/hashicorp/terraform/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!```

**Comment by sougat818** [comment]
I tried debugging this issue and when I build the module from master the issue was resolved. Please release a version :)

**Comment by Mongey** [comment]
Sorry for the late reply. Hopefully this was fixed in the `0.0.3` / `0.0.4`
